obs-studio-server/source/nodeobs_service.cpp

```diff
@@ -1973,6 +1977,17 @@ void OBS_service::setAudioSimpleRecordingEncoder(obs_encoder_t* encoder)
 	audioSimpleRecordingEncoder = encoder;
 }
 
+obs_encoder_t* OBS_service::getAudioAdvancedStreamingEncoder(void)
+{
+	return audioAdvancedStreamingEncoder;
+}
+
+void OBS_service::setAudioAdvancedStreamingEncoder(obs_encoder_t* encoder)
+{
+	obs_encoder_release(audioAdvancedStreamingEncoder);
```

Comment:
does obs_encoder_release handle NULL correctly in case audioAdvancedStreamingEncoder is NULL?

Reply:
Yes, actually I'm planning to remove all `if(encoder) release(encoder);` in the future, they are unnecessary since the function handles the null case.
Just gonna wait to remove they all together.